### PR TITLE
Channel authtype whitelist

### PR DESCRIPTION
**Pull request by BMartinos**
This PR relates to issue: https://github.com/jembi/openhim-console/issues/228

### CODE REVIEW

**Comment by BMartinos** [comment]
Hey @devcritter Do you mind just reviewing this PR for me when you have a chance. Thanks.

**Comment by BMartinos** [comment]
@debonair can you also maybe just glance over the code additions to make sure i haven't missed anything and it works as intended. Thanks

**Comment by debonair** [comment]
(thumbsup) looks good to me!

**Comment by BMartinos** [comment]
@devcritter had to make a small correction when channel is public. All completed now

File: app/views/partials/channels-tab-basic-info.html [hunk 1]
```diff
@@ -58,6 +58,21 @@
...
+  <label tooltip="Private requires authentication and Public bypasses authentication protocols">Authentication Type</label>
+  <div style="display: block;">
+    <div class="btn-group">
+      <label class="btn btn-primary" ng-model="channel.authType" btn-radio="'private'">Private</label>
+      <label class="btn btn-primary" ng-model="channel.authType" btn-radio="'public'">Public</label>
+    </div>  
+  </div>  
+</div>
+<div class="form-group">
+  <label tooltip="Enter the IP's that are allowed to access the channel">Whitelist IP's</label>
```
Comment (devcritter) [inline app/views/partials/channels-tab-basic-info.html]: super minor note: rather say 'IP addresses' instead of IPs, just for clarity.

**Comment by devcritter** [comment]
Just a super minor comment, but otherwise :+1:

**Comment by BMartinos** [comment]
@devcritter made that small change. Good to merge in?

**Comment by devcritter** [comment]
:+1: Go for it

**Pull request merged**